src/MotionModel/entities/conveyorBelt/conveyorBelt.cpp

```diff
@@ -0,0 +1,70 @@
+/*------------------------------- phasicFlow ---------------------------------
+      O        C enter of
+     O O       E ngineering and
+    O   O      M ultiscale modeling of
+   OOOOOOO     F luid flow       
+------------------------------------------------------------------------------
+  Copyright (C): www.cemf.ir
+  email: hamid.r.norouzi AT gmail.com
+------------------------------------------------------------------------------  
+Licence:
+  This file is part of phasicFlow code. It is a free software for simulating 
+  granular and multiphase flows. You can redistribute it and/or modify it under
+  the terms of GNU General Public License v3 or any other later versions. 
+ 
+  phasicFlow is distributed to help others in their research in the field of 
+  granular and multiphase flows, but WITHOUT ANY WARRANTY; without even the
+  implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+-----------------------------------------------------------------------------*/
+
+#include "conveyorBelt.hpp"
+#include "dictionary.hpp"
+
+
+FUNCTION_H
+pFlow::conveyorBelt::conveyorBelt(const dictionary& dict)
+{
+  if(!read(dict))
+	{
+		fatalErrorInFunction<<
+		"  error in reading conveyorBelt from dictionary "<< dict.globalName()<<endl;
+		fatalExit;
+	}
+}
+
+FUNCTION_H
+bool pFlow::conveyorBelt::read(const dictionary& dict)
+{	
+
+	velocity_ 		= dict.getVal<realx3>("velocity");
```

Comment:
change velocity_ to tangentVelocity_ here and in the header file (class declaration). 
here, the keyword "tangentVelocity" should be read from dictionary.